include/llvm/CodeGen/MachineModuleInfo.h

```diff
@@ -247,16 +244,6 @@ class MachineModuleInfo : public ImmutablePass {
   }
   /// \}
 
-  unsigned getBin(const Function *F) const {
-    auto I = Bins.find(F);
-    if (I != Bins.end())
-      return I->second;
-    else
-      return 0;
-  }
-  void setBin(const Function *F, unsigned Bin) {
-    Bins[F] = Bin;
-  }
```

Comment:
The LLVM folks will really appreciate (read as: demand  ;) that we keep the API changes/interactions minimal. Good changes overall!